opwen_email_server/config.py

```diff
@@ -18,3 +18,6 @@
 QUEUE_ERROR_FILE = environ.get('LOKOLE_QUEUE_ERROR_FILE', '')
 
 APPINSIGHTS_KEY = environ.get('LOKOLE_EMAIL_SERVER_APPINSIGHTS_KEY', '')
+
+MAX_WIDTH_IMAGES = int(environ.get('MAX_WIDTH_EMAIL_IMAGES', 200))
```

Comment:
Nit: `environ.get` always returns a string so the default value should also be a string. I.e., here have `'200'` instead of `200`. I'm surprised that the type-checker doesn't flag this since the type of `environ.get` should be `(str, Optional[str]) -> str`.

Same below.